crates/ignore/src/default_types.rs

```diff
@@ -167,7 +167,10 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
     ("pascal", &["*.pas", "*.dpr", "*.lpr", "*.pp", "*.inc"]),
     ("pdf", &["*.pdf"]),
     ("perl", &["*.perl", "*.pl", "*.PL", "*.plh", "*.plx", "*.pm", "*.t"]),
-    ("php", &["*.php", "*.php3", "*.php4", "*.php5", "*.phtml"]),
+    ("php", &[
+        "*.php", "*.php3", "*.php4", "*.php5", "*.php7", "*.php8",
+        "*.pht", "*.phtml"
```

Comment:
I don't think I've ever seen `pht` used for PHP before. Can you show me a project that uses it?

Reply:
It quite obscure. But I've just noticed that it's also used by other language eco-systems. Maybe we shouldn't include it to prevent overlaps and false-positives.

Reply:
[Github's search ](https://github.com/search?q=extension%3Apht) reveals some repos using it, e.g. https://github.com/chrisbroski/lulaleasuretime/tree/master/inc and https://github.com/RundizBones/ModuleAdmin/tree/3599dfbe41708dd94cfb7306dad1a44045004bd4/Console/CreateModuleTemplate

Reply:
Fair enough...